### PR TITLE
[TEVA-2554] do not show apply links for expired vacancy

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-2554

### CODE REVIEW

File: app/views/shared/vacancy/_jobseeker_view.html.slim [hunk 1]
```diff
@@ -26,7 +26,7 @@
               = render BannerButtonComponent.new text: t("jobseekers.job_applications.banner_links.submitted"),
                 href: jobseekers_job_application_path(job_application),
                 icon: "green-tick"
-          - else
+          - elsif @vacancy.expires_at.future?
```
Comment (cpjmcquillan) [inline app/views/shared/vacancy/_jobseeker_view.html.slim:29]: this is already true ⬆️ 

```ruby
- if JobseekerApplicationsFeature.enabled? && @vacancy.enable_job_applications? && @vacancy.expires_at.future?
```

File: app/components/jobseekers/vacancy_details_component/vacancy_details_component.html.slim [hunk 2]
```diff
@@ -32,18 +32,19 @@ section#job-details class="govuk-!-margin-bottom-5"
...
-    = govuk_link_to t("jobseekers.job_applications.apply"), new_jobseekers_job_job_application_path(@vacancy.id), button: true
-
-  - else
-    - if @vacancy.how_to_apply.present?
-      h4.govuk-heading-s = t("jobs.applying_for_the_job")
-      p = @vacancy.how_to_apply
-
-    - if @vacancy.application_link.present?
-      = govuk_link_to t("jobs.apply"), new_job_interest_path(@vacancy.id), target: "_blank", class: "vacancy-apply-link govuk-!-margin-bottom-5", button: true, "aria-label": t("jobs.aria_labels.apply_link")
+  - if @vacancy.expires_at.future?
```
Comment (cpjmcquillan) [inline app/components/jobseekers/vacancy_details_component/vacancy_details_component.html.slim:35]: Strictly speaking the ticket only talks about not displaying the `Apply for this job` button (our application journey) for expired vacancies. We already show the `More on how to apply` button for expired vacancies. 

Can you clarify this?

**Review by cpjmcquillan — APPROVED** [review]

File: app/components/jobseekers/vacancy_details_component/vacancy_details_component.html.slim (hunk 2 above)
Comment (alexbowen) [reply app/components/jobseekers/vacancy_details_component/vacancy_details_component.html.slim:35]: @bencmitchell  clarified that all the stuff not being rendered for expired in this PR is correct - he will comment on jira

**Comment by bencmitchell** [comment]
Checked an expired listing, both apply links removed so looks good.

**Pull request merged**